### PR TITLE
Feature 19

### PR DESCRIPTION
**Pull request by LeoCode0**
## Checklist

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [x] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [ ] Squash (default)
- [x] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Purpose

- I create the next buttons to get authentication:
  - login
  - sign up
  - log out
  - Authenticated button (This button choose between login or signup to show to users)

### Proposal

We need to change a little our application flow, the reason is that we have to leave from the to make login or signup my proposal is create a button (login, signup or log out) under the profile pic. 

### Relevant tickets
resolves [#19 ]

### Screenshots
Screenshot 2022-01-23 at 01-05-21 GETHIRED Find your dream job

### CODE REVIEW

**Comment by LeoCode0** [comment]
Also, now we have a .env file to add environment variables, currently I added API, ClientID and domainId  💯

**Comment by odagora** [comment]
Ok so I tested and I have two questions:

1. I imagine we need to fill in the .env values in order for it to work, isn't it?. I pulled the changes and reinstalled node_modules but when I click on the login button it shows the error: `undefined’s DNS address could not be found`
2. The Profile page you added is accesible via any specific route?

**Review by ZajithCorro — COMMENTED** [review]
I couldn't see what you did beacuse of this:
Captura de error

I added the domain and the api in the .env file, but it didn't work 😐

File: src/components/AuthButtons/index.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    return (
+        <Button variant="contained" type="button" onClick={() => loginWithRedirect()}>
+            Log in
+        </Button>
+    );
+};
+
+export const SignupButton = () => {
+    const { loginWithRedirect } = useAuth0();
```
Comment (odagora) [inline src/components/AuthButtons/index.jsx:17]: Al presionar el botón me sale un error `404 not found`. Analizando el tema, hay un `/` que genera el error en la url de redirect. Si la quito manualmente funciona (ver imagen):

**Review by odagora — APPROVED** [review]

**Pull request merged**